pkg/operator/ceph/csi/template/rbd/csi-rbdplugin-holder.yaml

```diff
@@ -0,0 +1,67 @@
+# This DaemonSet is only active when the cluster is deployed with Multus enabled or EnableCSIHostNetwork
```

Comment:
Looks like these holder pods were mistakenly added back in after the rebase

Reply:
oops, will fix